clang/unittests/Interpreter/CMakeLists.txt

```diff
@@ -26,6 +27,20 @@ add_distinct_clang_unittest(ClangReplInterpreterTests
   TargetParser
   )
 
+if(TARGET compiler-rt)
+  list(APPEND CLANG_REPL_TEST_SOURCES
+    OutOfProcessInterpreterTests.cpp
+  )
+  add_dependencies(ClangReplInterpreterTests 
+    llvm-jitlink-executor 
+    compiler-rt
+  )
+  message(STATUS "Compiler-RT found, enabling OOP JIT tests")
+  set(CLANG_HAS_COMPILER_RT TRUE)
+else()
+  set(CLANG_HAS_COMPILER_RT FALSE)
```

Comment:
We don't seem to be using these anywhere not that we conditionally include the test case.